lib/wsapi/session.rb

```diff
@@ -44,20 +44,37 @@ def initialize(session_id, opts = {})
       @conn = connection(session_id)
     end
 
-    def setup_refresh_token(client_id, client_secret, refresh_token, &block)
-      @oauth2 = {
-        client_id: client_id,
-        client_secret: client_secret,
-        refresh_token: refresh_token,
-        refresh_token_updated: block
-      }
+    def build_object_url(type, id)
+      wsapi_resource_url("#{type}/#{id}").gsub('userstory', 'hierarchicalrequirement')
     end
 
-    def get_user_subscription
-      response = wsapi_get(wsapi_resource_url("Subscription"))
+    def create(type, fields)
+      response = wsapi_request(:put, wsapi_resource_url("#{type}/create"), {type => fields})
```

Comment:
I think they're basically doing the same thing. Good eye. Not sure if there's a reason to change for "consistency" or not.